harness/tests/common/api_server.py

```diff
@@ -29,9 +29,25 @@
 FIXTURES_DIR = Path(__file__).parent.parent / "fixtures"
 
 
-def sample_get_experiment() -> bindings.v1GetExperimentResponse:
+def sample_get_experiment(**kwargs: Any) -> bindings.v1GetExperimentResponse:
```

Comment:
can we just delete this file and move this method into a utils of some sort?

Reply:
OOB, chatted about a `tests.fixtures.api_responses`.